app/translations/messages.fr.yml

```
@@ -609,12 +609,12 @@ Entities:
   LdapConnector:
     labels:
     actions:
-      add: "Ajouter un connecteur Ldap"
-      edit: "Editer le connecteur Ldap"
+      add: "Ajouter un connecteur LDAP"
+      edit: "Editer le connecteur LDAP"
```

C’était pas dans le ticket, mais ça sera mieux :

```suggestion
      edit: "Modifier le connecteur LDAP"
```

(le sens de "éditer" est plus proche de "publier", même si en informatique on le pige souvent comme la francisation de "edit")